src/sdk-impl.js

```diff
@@ -358,10 +358,15 @@ SdkImpl.prototype.onAdError = function(adErrorEvent) {
       adErrorEvent.getError !== undefined ?
           adErrorEvent.getError() : adErrorEvent.stack;
   window.console.warn('Ad error: ' + errorMessage);
+
   this.adsManager.destroy();
   this.controller.onAdError(adErrorEvent);
-};
 
+  // reset these so consumers don't think we are still in an ad break, but reset them after
```

Comment:
Can you split this up so that this line is 80 chars or less

Reply:
@ypavlotsky updated, sorry about that.